### PR TITLE
closes #76 Add create item function for admin

### LINKED ISSUE #76: Admin Item Creation
As an authenticated Admin:
I can create an item.
  - An item must have a title, description and price.
  - An item must belong to at least one category.
  - The title and description cannot be empty.
  - The title must be unique for all items in the system.
  - The price must be a valid decimal numeric value and greater than zero.
  - The photo is optional. If not present, a stand-in photo is used.

### PR DESCRIPTION
**Pull request by Tman22**
Admin can choose artist and category while creating item.
Also added a javascript overwrite in layout for materialize.
This allowed me to show the drop bars in new_admin_item_path.

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Admin::ItemsController < Admin::BaseController
```
Comment (ToniRib) [inline app/controllers/admin/items_controller.rb]: I'm going to keep ignoring Hound on this one.

**Comment by ToniRib** [comment]
Looks good overall, @Tman22 !! I'm adding some model tests to cover some of the finer points of the user story, like titles having to be unique. I also cleaned up the feature test you wrote to use our new factory girl factories for artist and admin.

**Comment by ToniRib** [comment]
Alright I made a few changes:
  - Added model tests for item creation (photo optional, only prices greater than 0 that are numbers), title has to be unique
  - Default image is a "photo not available" that is set through a before_save callback if the admin didn't specify an image
  - Added required: true to all the admin new item form fields except image_path
  - Updated feature test to use factory girl to create artist & admin
  - Updated navbar to display '(admin)' next to 'Logged in as <username>' so that it's easier to tell if an admin is logged in. Also, clicking this will redirect an admin to their admin_dashboard instead of regular dashboard
  - cleaned up the hound complaints

Let me know what you think @Tman22 & @martensonbj

**Comment by Tman22** [comment]
Great model tests Toni! Looks great!

**Comment by Tman22** [comment]
Let's merge this in now to master?

**Comment by ToniRib** [comment]
Will do!

**Pull request merged**